app/views/api/activities/index.rabl

```diff
@@ -1,6 +1,6 @@
 collection @activities
 
-attribute :id, :name, :location, :price, :show_on_website, :description
+attribute :id, :name, :location, :price, :show_on_website, :description_nl, :description_en
```

Comment:
Probably the activity section on the site won't work after this change, we should defs have PR ready in static-sticky

Reply:
Did that! Available at https://github.com/svsticky/static-sticky/pull/236